payments/admin.py

```diff
@@ -1,6 +1,29 @@
+import json
+
 from django.contrib import admin
-from payments.models import StripeAccount, StripeEvent
+from .models import StripeAccount, StripeEvent
+
+
+class StripeAccountAdmin(admin.ModelAdmin):
+
+    readonly_fields = (
+        'user', 'account_id', 'secret_key', 'public_key',
+        'available_balance'
+    )
+
+
+class StripeEventAdmin(admin.ModelAdmin):
+    list_display = (
+        'event_id', 'type', 'created', 'verified', 'processed', 'user_id')
+    search_fields = ['message_text']
+
+    readonly_fields = (
+        'type', 'created', 'event_id', 'processed', 'message_text',)
 
+    def message_text(self, instance):
```

Comment:
The manage user secret key is only sent when the account is created.  Pretty sure it is never sent with an event message.  From https://stripe.com/docs/connect/managed-accounts:

"You’ll want to store the account ID and, if you intend to use them for authentication, the secret key and publishable key. The keys will not be retrievable subsequently."